### ISSUE TITLE
Implement Multi-slot Inventory

### ISSUE THREAD

**Issue by simon987** [simon987/Much-Assembly-Required#133]
[CubotInventory](https://github.com/simon987/Much-Assembly-Required/blob/master/Plugin%20Cubot/src/main/java/net/simon987/cubotplugin/CubotInventory.java) should have a configurable amount of slots. The individual slots should be accessed by specifying an additional argument

**Comment by Chris211313** [comment]
Would love to get involved. Can I grab this issue?

**Comment by simon987** [comment]
Yes that would be awesome, let me know if you have any questions (preferably in the slack channel). If you want an easier time setting up the development environment, you can use the `spark` branch.

**Comment by Ayush-Kori** [comment]
Hey I want to contribute to this issue can u guide me how to solve this issue ??
